islandora_scholar.install

```diff
@@ -149,3 +149,15 @@ function islandora_scholar_update_7103(&$sandbox) {
   $t = get_t();
```

Comment:
Don't we need to move the variables from islandora_google_scholar.install into islandora_scholar.install?

Reply:
I believe they are already there.

Reply:
Egad, you're right! They're in both files:

https://github.com/Islandora/islandora_scholar/blob/7.x/islandora_scholar.install#L45-L47
https://github.com/Islandora/islandora_scholar/blob/7.x/modules/islandora_google_scholar/islandora_google_scholar.install#L13-L15

That's... That's bad, right?